quickstart/201-azfw-with-secure-hub/provider.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.69.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false // Set to True for Production
```

Comment:
Is there a reason to set this to false?

Reply:
can remove if needed.

Reply:
got it thanks Tom, i updated it with the virtual machine feature.